Directory: ./sample35
Test Output:
============================= test session starts ==============================
platform darwin -- Python 3.10.12, pytest-7.4.0, pluggy-1.3.0
rootdir: /Users/DELL/Desktop/new_chatgpt/cat3_classes
plugins: dash-2.14.0
collected 0 items / 1 error

==================================== ERRORS ====================================
__________________ ERROR collecting sample35/test_example.py ___________________
ImportError while importing test module '/Users/DELL/Desktop/new_chatgpt/cat3_classes/sample35/test_example.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../../opt/anaconda3/envs/pyng310/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
sample35/test_example.py:3: in <module>
    from example import NumpydocParser
sample35/example.py:11: in <module>
    from common import (
E   ModuleNotFoundError: No module named 'common'
=========================== short test summary info ============================
ERROR sample35/test_example.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.12s ===============================

Test Error Output:


